src/QueryHandler/QueryHandler.php

```diff
@@ -98,7 +91,7 @@ public function query(GraphQLSchema $schema, $query, ?array $vars = []): array
      * @param array|null $vars
      * @return array|ExecutionResult
      */
-    public function queryAndReturnResult(GraphQLSchema $schema, $query, ?array $vars = [])
+    public function queryAndReturnResult(GraphQLSchema $schema, string|DocumentNode $query, ?array $vars = []): array|ExecutionResult
```

Comment:
Not supported in PHP 7.4
```suggestion
    public function queryAndReturnResult(GraphQLSchema $schema, $query, ?array $vars = [])
```